api/swagger.yaml

```diff
@@ -864,9 +862,7 @@ definitions:
           The entry point for the container as a string or an array of strings.
```

Comment:
We'd probably have to update the description a bit to mention that it's an array of strings, but accepts a string for backward compatibility

ping @dnephin - can you confirm it's only for backward compatibility, or is there still a code path that actually sends a string instead of an array?

Reply:
As far as I can tell the CLI has not sent a single string in a very long time so it should be fine to only document this as an array.